bin/setup

```diff
@@ -34,6 +34,8 @@ Dir.chdir APP_ROOT do
   run "cp keys/equifax_rsa.example keys/equifax_rsa"
   run "cp keys/equifax_rsa.pub.example keys/equifax_rsa.pub"
   run "cp certs/saml.crt.example certs/saml.crt"
+  run "gpg --dearmor < keys/equifax_gpg.pub.example > keys/equifax_gpg.pub.bin"
+  system "gpg --batch --import keys/equifax_gpg.example"
```

Comment:
ah! that would make sense, for verifying in the tests then. Guess my old self was right.

+1 to leave it there then.